lib/fog/profitbricks/requests/compute/get_all_lans.rb

```diff
@@ -93,6 +93,8 @@ def get_all_lans(datacenter_id)
                         :method  => 'GET',
                         :path    => "/datacenters/#{datacenter_id}/lans?depth=1"
                     )
+                rescue => error
+                    Fog::Errors::NotFound.new(error)
```

Comment:
The documentation does not states what happens when no lans are encountered but i guess it does return an empty hash, right?

If so why would we need to rescue exceptions here and thrown a not find error? Wouldn't that be more confusing?